rayon-core/src/lib.rs

```diff
@@ -532,6 +537,34 @@ impl<S> ThreadPoolBuilder<S> {
         self
     }
 
+    /// Use the current thread as one of the threads in the pool.
+    ///
+    /// The current thread is guaranteed to be at index 0, and given the thread is not managed by
+    /// rayon, so spawn and exit handlers do not run for that thread.
```

Comment:
```suggestion
    /// The current thread is guaranteed to be at index 0, and since the thread is not managed by
    /// rayon, the spawn and exit handlers do not run for that thread.
```

Reply:
Done, thanks! Also rebased :)